crd/config/crds/networking.gke.io_networks.yaml

```diff
@@ -70,6 +70,13 @@ spec:
                 description: Gateway4 defines the gateway IPv4 address for the network.
                   Required if ExternalDHCP4 is false or not set on L2 type network.
                 type: string
+              ipam4:
```

Comment:
if you don't add this to v1alpha1 as well, round-trip read/write via v1alpha1 will drop values from this field

Reply:
don't you need validation in place that ensures IPAMMode and ipam4 aren't set at the same time?